### PR TITLE
renamed RuleFile into PrometheusRule, as well as ruleFileSelector into ruleSelector to reflect latest changes

### PR DESCRIPTION
**Pull request by sichvoge**

### CODE REVIEW

File: Documentation/user-guides/alerting.md [hunk 1]
```diff
@@ -121,22 +121,22 @@ spec:
...
+Prometheus rule files are held in `PrometheusRule` custom resources. Use the label selector field `ruleSelector` in the Prometheus object to define the rule files that you want to be mounted into Prometheus.
 
-The best practice is to label the `RuleFile`s containing rule files with `role: alert-rules` as well as the name of the Prometheus object, `prometheus: example` in this case.
+The best practice is to label the `PrometheusRule`s containing rule files with `role: alert-rules` as well as the name of the Prometheus object, `prometheus: example` in this case.
 
 [embedmd]:# (../../example/user-guides/alerting/prometheus-example-rules.yaml)
 ```yaml
 apiVersion: monitoring.coreos.com/v1
-kind: RuleFile
+kind: PrometheusRule
```
Comment (brancz) [inline Documentation/user-guides/alerting.md]: The `[embedmd]` headers before each of these yaml blocks, indicate, that this is embedded by the file referenced. You need to change that file accordingly, or CI will fail.

Comment (mxinden) [reply Documentation/user-guides/alerting.md]: After changing the referenced file, simply run `make generate`, which will update the section in each markdown file.

**Comment by mxinden** [comment]
CI status for this PR can be found here: https://travis-ci.org/coreos/prometheus-operator/builds/391298440

(Never seen Travis failing to set it on the bottom of a Github PR before :confused: )

**Comment by brancz** [comment]
@sichvoge could you just squash this PR and re-trigger CI?

**Pull request closed without merging**